nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -1305,7 +1305,7 @@ protected synchronized void rollback(final boolean penalize, final boolean rollb
         try {
             context.getFlowFileEventRepository().updateRepository(flowFileEvent, connectable.getIdentifier());
         } catch (final Exception e) {
-            LOG.error("Failed to update FlowFileEvent Repository due to " + e);
+            LOG.error("Failed to update FlowFileEvent Repository", e);
             if (LOG.isDebugEnabled()) {
                 LOG.error("", e);
             }
```

Comment:
The previous behavior only printed the stack trace when debug was enabled. Making this change to just log the error with the stack trace is a cleaner way to go from a code perspective, and current log handling should provide sufficient control over stack traces. As a result, this type of conditional can be removed.
```suggestion
```